CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix a bug where parsing a `ChipFamily` from source would always fail.
```

Comment:
This is not really clear to me, from this I don't understand what a `ChipFamiliy` is, and why or when it is parsed from source. Maybe mention that this is used to read a chip definition from a YAML file?